src/http/chain-registry/ChainRegistry.test.ts

```diff
@@ -0,0 +1,183 @@
+import { HttpClient } from '../client/http'
+import { IChain, IHttpGETRequest, IHttpPOSTRequest, IRegisterChainRequest } from '../types'
+
+import { ChainRegistry } from './ChainRegistry'
+
+const mockHTTPClient = {
+  get: jest.fn(),
+  post: jest.fn()
+}
+
+jest.mock('../client/http', () => ({
```

Comment:
Please use jest.mock only when mocking external packages. For internal classes I believe you can use `jest-create-mock-instance` . See example on `ResponseMessage.test.ts`

Reply:
I think in this I cannot as I am trying to mock the import of `../client/http.js` happening at `new FaucetRegistry()` and `new ChainRegistry()`